tests/integration/pilot/nodelocal/main_test.go

```diff
@@ -0,0 +1,55 @@
+//go:build integ
+// +build integ
+
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nodelocal
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/resource"
+	"istio.io/istio/tests/integration/pilot/common"
+)
+
+var (
+	i istio.Instance
+
+	// Below are various preconfigured echo deployments. Whenever possible, tests should utilize these
+	// to avoid excessive creation/tear down of deployments. In general, a test should only deploy echo if
+	// its doing something unique to that specific test.
+	apps = &common.EchoDeployments{}
+)
+
+// TestMain defines the entrypoint for pilot tests using a standard Istio installation.
+// just use for this test.
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite(m).
```

Comment:
I use RequireMinClusters(2) and RequireMinVersion(21) want to create two cluster test this feature, but the CI log find that only one cluster created. How can I configure it.

integ-pilot-k8s-tests_istio #1432877024984174592 log：
https://storage.googleapis.com/istio-prow/pr-logs/pull/istio_istio/34457/integ-pilot-k8s-tests_istio/1432877024984174592/build-log.txt
```
2021-09-01T01:46:18.157042Z	info	tf	=== Test Framework Settings ===
2021-09-01T01:46:18.157126Z	info	tf	TestID:            pilot_nodelocal
RunID:             c71da29c-9c00-482a-90f1-8822ff0634e3
NoCleanup:         false
BaseDir:           /logs/artifacts
Selector:          -flaky,-postsubmit
FailOnDeprecation: false
CIMode:            true
Retries:           0
StableNamespaces:  false
Revision:          
SkipVM:            false
Compatibility:     false
Revisions:         

2021-09-01T01:46:18.157131Z	info	tf	===============================
2021-09-01T01:46:18.157300Z	info	tf	Test run dir: /logs/artifacts/pilot-nodelocal-c71da29c9c00482
2021-09-01T01:46:18.157426Z	info	tf	Test Framework Kubernetes environment Settings:
Kubeconfigs:           [/home/.kube/config]
LoadBalancerSupported:      true
ControlPlaneTopology: map[]
NetworkTopology:      map[]
ConfigTopology:      map[]

2021-09-01T01:46:18.157442Z	info	tf	Using KubeConfigs: [/home/.kube/config].
2021-09-01T01:46:18.157456Z	info	tf	=== BEGIN: Building clusters ===
2021-09-01T01:46:18.160390Z	info	tf	Built Cluster:
Name:               cluster-0
StableName:         primary-0
Kind:               Kubernetes
PrimaryCluster:     cluster-0
ConfigCluster:      cluster-0
Network:            
HTTPProxy:            
Filename:           /home/.kube/config

2021-09-01T01:46:18.160424Z	info	tf	=== DONE: Building clusters ===
2021-09-01T01:46:18.160435Z	info	tf	=== BEGIN: Setup: 'pilot_nodelocal' ===
2021-09-01T01:46:18.160445Z	info	tf	Skipping suite "pilot_nodelocal": Number of clusters 1 does not exceed minimum 2
=== RUN   TestNodeLocal
    test.go:213: Skipped because parent Suite was skipped.
--- SKIP: TestNodeLocal (0.00s)
PASS
```